docs/content/en/docs/yaml-crd-ref/metric.md

```diff
@@ -4,11 +4,7 @@ description: Define all workloads and checks associated with an application
 weight: 50
 ---
 
-A `KeptnMetric` represents a metric that is collected from a provider.
-Providing the metrics as a CR in a Kubernetes cluster
-facilitates the reusability of this data across multiple components
-and allows using multiple observability platforms
-for different metrics at the same time.
+A `KeptnMetric` is a way to capture and share data from a metric source. By creating a Custom Resource (CR) in a Kubernetes cluster to represent these metrics, it makes it easier to reuse this data across various components. This approach also enables the use of different observability platforms for various metrics simultaneously.
```

Comment:
Need to break into multiple lines.  But this very verbose description seems to be more about the whole metrics use case and metrics-operator, not the KeptnMetric resource per se.